Dockerfile

```
@@ -4,14 +4,22 @@ COPY package.json package-lock.json* ./
+RUN npm ci
 
 FROM node:22-trixie-slim AS build
 WORKDIR /app
 COPY --from=deps /app/node_modules ./node_modules
 COPY . .
 RUN npm run build
 
+FROM node:22-trixie-slim AS production-deps
+WORKDIR /app
+COPY package.json package-lock.json* ./
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)